iMessageClone/iMessageChatChannelListItemView.swift

```diff
@@ -24,34 +24,15 @@ final class iMessageChatChannelListItemView: ChatChannelListItemView {
     
     override func setUpLayout() {
         super.setUpLayout()
-        
+
+        // Add left unread view blue dot.
         unreadView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(unreadView)
```

Comment:
Why not insert this as the 1st view of the main container?

Reply:
FIxed